src/hotspot/cpu/riscv/stubRoutines_riscv.cpp

```diff
@@ -55,4 +55,13 @@ address StubRoutines::riscv::_string_indexof_linear_ul = nullptr;
 address StubRoutines::riscv::_large_byte_array_inflate = nullptr;
 address StubRoutines::riscv::_method_entry_barrier = nullptr;
 
+ATTRIBUTE_ALIGNED(64) const jint StubRoutines::riscv::_arrays_hashcode_powers_of_31[] =
```

Comment:
If we use `mv` to replace the `ld` of power values above, these related code could be removed here.